### PR TITLE
Add aria label to announce VAOS alert message

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
Add Aria label to announce facility alert message for accessibility

## Testing done 
\[Provide link to QA Test Plan ticket (issue-template coming soon) for the relevant product/feature.
List the new/update unit-test(s) & e2e-test\(s) that have been successfully run before open this PR.
Also, if UI-testable (i.e., testable manually on Staging by QA, either via the browser or REST API-client), provide link to QA test-request ticket \[issue-template coming soon].]

See [VSA QA Process](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/qa/vsa-qa-process.md) for more info on Engineering/QA collaboration/engagement.

## Screenshots

image

## Acceptance criteria
- [ ] add aria-atomic="true" aria-live="assertive" for screen reader

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the Pre-Launch Checklist
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] A link has been provided to the VSA-QA Test Plan ticket \[use the VSA-QA Test Plan Issue Template to create the ticket within va.gov-team repo].
- [ ] A link has been provided to the VSA-QA Test-Request ticket \[issue-template coming soon].
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/components/VAFacilityInfoMessage.jsx [hunk 1]
```diff
@@ -3,20 +3,23 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
...
-      Not all VA locations offer all types of care or support online scheduling.
-      <p>
-        If this location wasn't what you were looking for, you can{' '}
-        <a href="/find-locations" target="_blank" rel="noopener noreferrer">
-          search for a nearby location
-        </a>{' '}
-        and call to schedule an appointment.
-      </p>
-    </AlertBox>
+    <div aria-atomic="true" aria-live="assertive">
```
Comment (jbalboni) [inline src/applications/vaos/components/VAFacilityInfoMessage.jsx]: I don't think we want to add these to this message. It's informational and is the primary page content.

Comment (jenniemc) [reply src/applications/vaos/components/VAFacilityInfoMessage.jsx]: @jbalboni, i'll revert VAFacilityInfoMessage.jsx back to previous version and remove VAFacilityInfoMessage.unit.spec.jsx from the commit.  Is there any others that need to revert back?

Comment (jbalboni) [reply src/applications/vaos/components/VAFacilityInfoMessage.jsx]: @jenniemc Nope, just this one.

Comment (jenniemc) [reply src/applications/vaos/components/VAFacilityInfoMessage.jsx]: @jbalboni, this commit failed the check in the Jenkin pipeline at Integration. Can you take a look why it failed at Accessibility?

**Review by jbalboni — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]

**Pull request merged**